Libraries/LibJS/Tests/empty-statements.js

```diff
@@ -1,11 +1,19 @@
-try {
-    ;;;
-    if (true);
-    if (false); else if (false); else;
-    while (false);
-    do; while (false);
-
-    console.log("PASS");
-} catch (e) {
-    console.log("FAIL: " + e);
-}
+test("empty semicolon statements", () => {
+    expect(";;;").toCompile();
+});
+
+test("if with no body", () => {
+    expect("if (true);").toCompile();
+});
+
+test("chained ifs with no bodies", () => {
+    expect("if (false); else if (false); else;").toCompile();
+});
+
+test("while with no body", () => {
+    expect("while (false);").toCompile();
+});
+
+test("do while with no body", () => {
+    expect("do; while (false);").toCompile();
+});
```

Comment:
Same here, behaviour changes from actually running the stuff to "just" parsing it.

Reply:
...this might or might not be what we want, open to suggestions :)